examples/evil_lualine.lua

```diff
@@ -160,7 +160,7 @@ ins_left {
   function()
     local msg = 'No Active Lsp'
     local buf_ft = vim.api.nvim_buf_get_option(0, 'filetype')
```

Comment:
vim.api.nvim_buf_get_option is also deprecated with nvim 0.10.
I believe that `local buf_ft = vim.api.nvim_get_option_value('filetype', { buf = 0 })
` should be used instead

Reply:
For better readability and be more specific defining the buffer, we can use `local buf_ft = vim.api.nvim_get_option_value('filetype', { 'buf': 0 })`, no?
The scope option it's not necessary because by default the use of buf implies {scope} is `local`.
Here is the [documentation](https://neovim.io/doc/user/api.html#nvim_get_option_value())

Reply:
Yeah the only adjustment is instead of `'buf' : 0` it should be `buf = 0`